### PR TITLE
Schedule detail screen(live screen) - ライブ面談画面

### PR DESCRIPTION
**Pull request by furuhiro-jp**
issue: https://github.com/dank1ntq/app/issues/8

## Design
スクリーンショット 2019-03-28 17 34 19

## Create screen
live

@HungDoNTQ Please review.

### CODE REVIEW

File: src/components/Live.vue [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+      return this.status === LIVE_INCOMING_CALL
+    },
+    isJoining () {
+      return this.status === LIVE_JOINING
+    },
+    pageTitle () {
+      if (this.isWaiting) {
+        return 'LIVE waiting'
+      }
+      return ''
```
Comment (ntqhungdx) [inline src/components/Live.vue]: Can edit to `return this.isWaiting ? 'LIVE waiting' : ''`.

File: src/components/Live.vue [hunk 2]
```diff
@@ -0,0 +1,325 @@
...
+    },
+    mainClassName () {
+      if (this.isWaiting) {
+        return 'screen-box__main'
+      }
+      return ''
```
Comment (ntqhungdx) [inline src/components/Live.vue]: As above

File: src/components/Live.vue (hunk 1 above)
Comment (furuhiro-jp) [reply src/components/Live.vue]: Thank you for pointing out.
It is easy to see this one. to correct.

**Comment by furuhiro-jp** [comment]
@ntqhungdx Fixed. Please review again.

**Pull request merged**